### PR TITLE
Refactor interview

### PR DESCRIPTION
**Pull request by olivierschultz**
- Restructure preparation
- Add behavioral questions
- Fix typo

### CODE REVIEW

**Comment by olivierschultz** [comment]
@BerthelotAnthony Can you check this one aswell please? You can rebase and merge when you're done also. Ty

**Comment by BerthelotAnthony** [comment]
@olivierschultz Sure.

File: interview/interview_cover_letter.md [hunk 1]
```diff
@@ -52,6 +51,6 @@ __Tip__: Writing a reason like, "I am interested in working for a top company" i
```
Comment (BerthelotAnthony) [inline interview/interview_cover_letter.md]: inco**n**sistencies

File: interview/interview_linkedin.md [hunk 2]
```diff
@@ -1,76 +1,75 @@
...
+- [ ] My Linkedin profile includes a list of skills which are gained from my past experience and which are relevant to the job I wish to have.
 
-# Connections/Network
+## Connections/Network
 
-- [x] My Linkedin profile has at least 50 connections.
+- [ ] My Linkedin profile has at least 50 connections.
 
-- [x] My Linkedin profile demonstrates that I have joined relevant groups to the position/indistry I wish to join.
+- [ ] My Linkedin profile demonstrates that I have joined relevant groups to the position/indistry I wish to join.
```
Comment (BerthelotAnthony) [inline interview/interview_linkedin.md]: indistry ? industry? maybe company ?

**Comment by BerthelotAnthony** [comment]
@olivierschultz I will finish the review later today

**Pull request merged**